sirepo/server.py

```diff
@@ -406,7 +413,7 @@ def api_newSimulation(self):
             req.template.new_simulation(d, req.req_data)
         return self._save_new_and_reply(req, d)
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec("require_user", sid="SimId", model="Model", title="Title")
```

Comment:
model=ComputeModelName optional, title=DownloadNamePostfix optional, sid has to be simulation_id to match args.